src/scripts/btma/annotator.py

```diff
@@ -30,12 +30,14 @@
 script_path = path.Path(__file__).parent.parent.absolute()
 
 omn_path = src_path / 'ontology' / 'edits'              
-csv_path = src_path / 'ontology' / 'exports'          
+csv_path = script_path / 'btma' / 'data'          
+db_path = script_path / 'btma' / 'database'
+exp_path = script_path / 'btma' / 'exports'
 owl_path = src_path / 'ontology' / 'imports'
 robot_path = script_path / 'btma' / 'robot.jar'
 
-def extract_IDs(module : list) -> list:
-    file = open(csv_path / (module + '.csv'))
+def extract_IDs(path) -> list:
+    file = open(path)
```

Comment:
better use the formulation `with open(path) as file: ...` so that the file is automatically closed again afterwards.

Reply:
Done